mslicedevel.bat

```diff
@@ -7,9 +7,5 @@
 @set PYTHON_EXE=C:\MantidInstall\bin\python.exe
 @set QT_API=pyqt
 
-:: Build and run mslice
-%PYTHON_EXE% %THIS_DIR%setup.py build_py --inplace
-@if %ERRORLEVEL% NEQ 0 (
-  exit /B %ERRORLEVEL%
 )
```

Comment:
We've got a stray `(` left.

Can you also remove the corresponding lines for the linux shell script?